assets/xml/objects/object_lightblock.xml

```diff
@@ -1,9 +1,10 @@
 ﻿<Root>
+    <!-- Object for Lightblock -->
```

Comment:
```suggestion
    <!-- Object for the Sun Block -->
```
It's better to try and describe what this is in-game instead of just copying the file name